### PR TITLE
[PROF-10209] Update docs for python timeline

### PR DESCRIPTION
**Pull request by taegyunkim**
### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

**Review by sanchda — APPROVED** [review]

File: content/en/profiler/connect_traces_and_profiles.md [hunk 1]
```diff
@@ -54,6 +54,12 @@ Code Hotspots identification is enabled by default when you [turn on profiling f
 
 Requires `dd-trace-py` version 0.44.0+.
 
+To enable the new [timeline feature](#span-execution-timeline-view)
```
Comment (cswatt) [inline content/en/profiler/connect_traces_and_profiles.md:57]:
```suggestion
To enable the [timeline feature](#span-execution-timeline-view):
```

File: content/en/profiler/connect_traces_and_profiles.md [hunk 2]
```diff
@@ -54,6 +54,12 @@ Code Hotspots identification is enabled by default when you [turn on profiling f
...
+
+- Upgrade `dd-trace-py` to version 2.10.4 or later
+- Set environment variable `DD_PROFILING_TIMELINE_ENABLED` to `true`
+- Set environment variable `DD_PROFILING_EXPORT_LIBDD_ENABLED` to `true`, this enables new profiling exporter implemented as a native extension.
```
Comment (cswatt) [inline content/en/profiler/connect_traces_and_profiles.md:61]:
```suggestion
- Set environment variable `DD_PROFILING_EXPORT_LIBDD_ENABLED` to `true`. This enables the profiling exporter to be implemented as a native extension.
```

Trying to fix grammar—does my change preserve the intended meaning of this sentence?

**Review by cswatt — DISMISSED** [review]
small edits

File: content/en/profiler/connect_traces_and_profiles.md (hunk 2 above)
Comment (taegyunkim) [reply content/en/profiler/connect_traces_and_profiles.md:61]: This bit is no longer needed.

**Comment by brett0000FF** [comment]
Hey! Just FYI I'm also updating the supported_versions page for the timeline view release: https://github.com/DataDog/documentation/pull/24625.

**Review by drichards-87 — APPROVED** [review]

**Pull request merged**